apm-agent-core/src/main/java/co/elastic/apm/agent/impl/stacktrace/StacktraceConfiguration.java

```diff
@@ -40,11 +40,24 @@
         .key(APPLICATION_PACKAGES)
         .configurationCategory(STACKTRACE_CATEGORY)
         .description("Used to determine whether a stack trace frame is an 'in-app frame' or a 'library frame'.\n" +
-            "Multiple packages can be set as a comma-separated list.\n" +
-            "Setting this option can also improve the startup time.\n" +
+            "This allows the UI to collapse the stack frames of library code,\n" +
+            "and highlight the stack frames that originate from your application.\n" +
+            "Multiple root packages can be set as a comma-separated list;\n" +
+            "there's no need to configure sub-packages.\n" +
+            "Because this setting helps determine which classes to scan on startup,\n" +
+            "setting this option can also improve startup time.\n" +
             "\n" +
-            "In order to be able to use the API annotations @CaptureTransaction and @CaptureSpan,\n" +
-            "it is required to set these options.")
+            "You must set this option in order to use the API annotations `@CaptureTransaction` and `@CaptureSpan`.\n" +
+            "\n" +
+            "**Example**\n" +
+            "\n" +
+            "Most Java projects have a root package, e.g. `com.myproject`.\n" +
+            "Here's how you can set that application package using Java system properties:\n" +
+            "\n" +
+            "[source,bash]\n" +
+            "----\n" +
+            "-Delastic.apm.application_packages=com.myproject\n" +
+            "----\n")
```

Comment:
For additional clarity, I would add something like:
```suggestion
            "And if you are only interested in specific subpackages, you can set something like:\n" + 
            `-Delastic.apm.application_packages=com.myproject.api,com.myproject.impl`\n")
```